CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if(BUILD_MULTI_GPU)
 endif()
 
 if(BUILD_PYT)
-  add_definitions(-D_GLIBCXX_USE_CXX11_ABI=1)
+  add_definitions(-D_GLIBCXX_USE_CXX11_ABI=0)
```

Comment:
why this change?

Reply:
Right now this parameter needs to be manually changed depending on the installed pytorch. Some pytorch were built with ABI:CXX11 (such as the pytorch package installed in the latest nvidia pytorch docker image), but some versions of pytorchs were not built with ABI:CXX11 (e.g., `1.10.1+cu113` built by torch team).

Reply:
make it an env variable?

Reply:
yes, we can setup it in cmake command.